sympy/functions/elementary/exponential.py

```diff
@@ -152,9 +152,9 @@ class exp_polar(ExpBase):
     is_polar = True
     is_comparable = False  # cannot be evalf'd
 
-    def _eval_Abs(self):
-        from sympy import expand_mul
-        return sqrt( expand_mul(self * self.conjugate()) )
+    def _eval_Abs(self):   # Abs is never a polar number
+        from sympy.functions.elementary.complexes import Abs
+        return Abs(exp(self.args[0]), evaluate=True)
```

Comment:
Is this better than `exp(re(self.args[0]))`?

Reply:
Probably not. I changed this.